app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorEffect.kt

```diff
@@ -1,7 +1,14 @@
 package org.simple.clinic.sync.indicator
 
+import java.util.concurrent.TimeUnit
+
 sealed class SyncIndicatorEffect
 
 object FetchLastSyncedStatus : SyncIndicatorEffect()
 
 object FetchDataForSyncIndicatorState : SyncIndicatorEffect()
+
+data class StartSyncedStateTimer(
+    val intervalAmount: Long,
+    val timeUnit: TimeUnit
```

Comment:
What do you think of replacing these two with a `Duration` instead and converting to the `TimeUnit` at the usage site?

We could also add an extension function on `Observable` that lets us set up an `interval` with `Duration`. We have done this for [`Observable#timer`](https://github.com/simpledotorg/simple-android/blob/master/app/src/main/java/org/simple/clinic/util/RxFoo.kt#L16).

Reply:
I will pick this in cleanup as well.